shared/ui/core/src/test/java/com/ivy/ui/FormatMoneyUseCaseTest.kt

```diff
@@ -24,79 +24,122 @@ class FormatMoneyUseCaseTest {
             shortenAmount = false,
+            isCrypto = false,
             locale = Locale.ENGLISH,
             expectedOutput = "1,000.12"
         ),
         ENG_HIDE_DECIMAL(
             amount = 1_000.12,
             showDecimal = false,
             shortenAmount = false,
+            isCrypto = false,
             locale = Locale.ENGLISH,
             expectedOutput = "1,000"
         ),
         GERMAN_SHOW_DECIMAL(
             amount = 1_000.12,
             showDecimal = true,
             shortenAmount = false,
+            isCrypto = false,
             locale = Locale.GERMAN,
             expectedOutput = "1.000,12"
         ),
         GERMAN_HIDE_DECIMAL(
             amount = 1_000.12,
             showDecimal = false,
             shortenAmount = false,
+            isCrypto = false,
             locale = Locale.GERMAN,
             expectedOutput = "1.000"
         ),
         ENGLISH_1K_SHORT_AMT(
             amount = 13_000.10,
             showDecimal = true,
             shortenAmount = true,
+            isCrypto = false,
             locale = Locale.ENGLISH,
             expectedOutput = "13k"
         ),
         ENGLISH_MILLION_SHORT_AMT(
             amount = 1_233_500.10,
             showDecimal = true,
             shortenAmount = true,
+            isCrypto = false,
             locale = Locale.ENGLISH,
             expectedOutput = "1.23m"
         ),
         ENGLISH_BILLION_SHORT_AMT(
             amount = 1_233_000_000.10,
             showDecimal = true,
             shortenAmount = true,
+            isCrypto = false,
             locale = Locale.ENGLISH,
             expectedOutput = "1.23b"
         ),
         GERMAN_1K_SHORT_AMT(
             amount = 13_000.10,
             showDecimal = true,
             shortenAmount = true,
+            isCrypto = false,
             locale = Locale.GERMAN,
             expectedOutput = "13k"
         ),
         GERMAN_MILLION_SHORT_AMT(
             amount = 1_233_500.10,
             showDecimal = true,
             shortenAmount = true,
+            isCrypto = false,
             locale = Locale.GERMAN,
             expectedOutput = "1,23m"
         ),
         GERMAN_BILLION_SHORT_AMT(
             amount = 1_233_000_000.10,
             showDecimal = true,
             shortenAmount = true,
+            isCrypto = false,
             locale = Locale.GERMAN,
             expectedOutput = "1,23b"
         ),
+        ENG_SHOW_DECIMAL_CRYPTO(
+            amount = 123_456.0,
+            showDecimal = true,
+            shortenAmount = false,
+            isCrypto = true,
+            locale = Locale.ENGLISH,
+            expectedOutput = "123456"
+        ),
+        ENG_HIDE_DECIMAL_CRYPTO(
+            amount = 123_456.0,
+            showDecimal = false,
+            shortenAmount = false,
+            isCrypto = true,
+            locale = Locale.ENGLISH,
+            expectedOutput = "123456"
+        ),
+        GERMAN_SHOW_DECIMAL_CRYPTO(
```

Comment:
We need a tear case for "0.000345 BTC" for example

Reply:
Done